package.json

```
@@ -90,7 +90,7 @@
     "yo/meow": "^9.0.0"
   },
   "pre-commit": [
-    "test-staged"
+    "yarn lint"
```

```suggestion
    "lint"
```

This should only need to be `lint`: https://www.npmjs.com/package/pre-commit#configuration